### PR TITLE
added dall.e.2.0

### PR DESCRIPTION
**Pull request by AmanSingh6574**
Issue #475

### CODE REVIEW

**Comment by JasonDsouza212** [comment]
The JS syntax is wrong , look into it and also add the tool at the last of the DB

**Comment by AmanSingh6574** [comment]
@JasonDsouza212 sir why there is this deployment error ?

**Comment by AmanSingh6574** [comment]
> The JS syntax is wrong , look into it and also add the tool at the last of the DB

sir not understood what exactly is wrong is it the description length? 10-12 words or something

File: src/DB/product.json [hunk 1]
```diff
@@ -1457,6 +1457,13 @@
   "link": "https://quillbot.com/",
   "description": "AI-powered paraphrasing tool that enhances your writing"
 },
+{
+  "productName": "DALL·E",
+  "category": "ai",
+  "image": "https://bit.ly/3oICzAN",
+  "link": "https://openai.com/dall-e-2",
+  "description": "An AI system that can create realistic images and art from a description in natural language."
+
```
Comment (shyamtawli) [inline src/DB/product.json:1466]:
```suggestion
},
```

**Review by shyamtawli — CHANGES_REQUESTED** [review]

**Comment by AmanSingh6574** [comment]
@JasonDsouza212 sir i have made the changes

**Pull request merged**